### ISSUE TITLE
Wrong indentation in padded-blocks

### ISSUE THREAD

**Issue by xbaun** [eslint/eslint#13547]
**Tell us about your environment**

* **ESLint Version: 7.6.0**
* **Node Version:**
* **npm Version:**

**What parser (default, Babel-ESLint, etc.) are you using?**
https://eslint.org/demo

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
    "parserOptions": {
        "ecmaVersion": 11,
        "sourceType": "script",
        "ecmaFeatures": {}
    },
    "rules": {
        "padded-blocks": 2
    },
    "env": {
        "es6": true,
        "browser": false,
        "es2017": true,
        "es2020": true,
        "webextensions": false
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

https://eslint.org/demo#eyJ0ZXh0IjoiY2xhc3MgRm9vIHtcblxuICBmb28oKSB7fVxuICBcbiAgYmFyKCkge1xuXHR0aGlzLmZvbygpO1xuICB9XG59Iiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMSwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6eyJwYWRkZWQtYmxvY2tzIjoyfSwiZW52Ijp7ImVzNiI6dHJ1ZSwiYnJvd3NlciI6ZmFsc2UsImVzMjAxNyI6dHJ1ZSwiZXMyMDIwIjp0cnVlLCJ3ZWJleHRlbnNpb25zIjpmYWxzZX19fQ==

**What did you expect to happen?**

The indentation of padded blocks should be correct.

**What actually happened? Please include the actual, raw output from ESLint.**

The closing curly brace of a block is not indented. Output:

```js
class Foo {

  foo() {}
  
  bar() {

	this.foo();
  
}

}
```

**Comment by anikethsaha** [comment]
Thanks for the issue

It looks like you need to use [indent](https://eslint.org/docs/rules/indent) as that rules checks for the indentation of the code.

**Comment by xbaun** [comment]
Thanks, didn't notice that I have to use ident rule. Solved the issue.
